botocore/compat.py

```diff
@@ -111,7 +111,11 @@ def set_socket_timeout(http_response, timeout):
     def accepts_kwargs(func):
         return inspect.getargspec(func)[2]
 
-    unicode = unicode
+    _unicode = unicode
+    def unicode(s, encoding='utf-8', errors='strict'):
```

Comment:
I'd prefer ensure_unicode instead of unicode given it doesn't quite behave the same as the built in unicode now.